.swiftpm/xcode/xcshareddata/xcschemes/AWSIntegrationTestsOnCI.xcscheme

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is basically a scheme with default settings, that delegates all test configuration to the test plan at `XCTestPlans/AWSIntegrationTestsOnCI.xctestplan`.